.github/workflows/release_notes.yml

```diff
@@ -50,4 +50,28 @@ jobs:
           pr_body: "This is an automated PR generated when the Ruby agent is released. Please merge as soon as possible."
           destination_repository: "newrelic/docs-website"
 
-# TODO: Add a job to delete branch on PR failure. 
+  delete_branch_on_fail:
+    name: Delete branch on fail
+    needs: [send-pull-requests]
+    runs-on: ubuntu-22.04
+    if: always()
+    steps:
```

Comment:
(as discussed offline) Ideally we don't want to run _any_ of these new 5 steps unless the workflow conclusion is "failure", but currently we'll perform steps 1-4 in ALL cases and then only guard the 5th step. This isn't a terribly big deal given the infrequency of the release notes workflow being called.

Reply:
This has been fixed by adding `if: failure()`